packages/backend-app-api/report.api.md

```diff
@@ -17,11 +18,30 @@ export interface Backend {
         }>,
   ): void;
   // (undocumented)
-  start(): Promise<void>;
+  start(): Promise<{
+    result: BackendStartupResult;
+  }>;
   // (undocumented)
   stop(): Promise<void>;
 }
 
+// @public
+export class BackendStartupError extends CustomErrorBase {
+  constructor(startupResult: BackendStartupResult);
+  // (undocumented)
+  name: 'BackendStartupError';
+  // (undocumented)
+  get result(): BackendStartupResult;
+}
+
+// @public
+export interface BackendStartupResult {
+  beginAt: Date;
+  outcome: 'success' | 'failure';
+  plugins: PluginStartupResult[];
```

Comment:
Does it make sense to actually cover for the service startup cycle too? Or do they always get wrapped up in the corresponding plugin/module startup even if they are global scope? What about eagerly initialized globals then?